### PR TITLE
Adds date to website

### PR DESCRIPTION
**Pull request by maxcell**
Yeah... It probably would be good to add this in lol. We should be able to add this before/after #24 and then get this as the result:

image

It also responses just fine with the CSS and so basically an easy add-on!

### CODE REVIEW

**Comment by stevenpetryk** [comment]
True! We'll want to render this in the proper timezone, though, so we have a few options:

* Make the timezone configurable from the environment, default to eastern:
  ```rb
  config.time_zone = ENV.fetch("TIMEZONE") { "Eastern Time (US & Canada)" }
  ```

* Render the time in React; the browser will automatically let moment select the correct timezone. This has the disadvantage of not showing up for non-JavaScript people.

**Comment by stevenpetryk** [comment]
Also, from this point on I hereby ban `<br />` from the codebase. Good luck! <3

File: app/views/countdowns/index.text.erb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<%= "%s" % @countdown.hackathon_starts_at.strftime('%B %dth @ %I:%M%p') %>
```
Comment (stevenpetryk) [inline app/views/countdowns/index.text.erb]: `<%= "%s" % string %>` is equivalent to just `<%= string %>`.

**Comment by maxcell** [comment]
I would like for us to keep the date displayed using Rails, noticed before but we never explicit said the date anywhere on the website already and I would enjoy if this was at least displayed if nothing else so everyone curious can see the information. So will update in just a second!

File: app/views/countdowns/index.text.erb (hunk 1 above)
Comment (maxcell) [inline app/views/countdowns/index.text.erb]: I appreciate it! For some reason I was being dumb about the partial stuff last night. I think I did it just to keep consistent to your formatting.

**Comment by maxcell** [comment]
Curious about the time configuration, with ISO8601, should we continue to have the `Z` designation in `T02:00:00Z` if we are trying to set the time zone in the environment?

File: app/views/countdowns/index.text.erb (hunk 1 above)
Comment (stevenpetryk) [inline app/views/countdowns/index.text.erb]: True, that's why I almost didn't even notice it at first!

**Comment by stevenpetryk** [comment]
You're completely right—we may just be able to change `ENV["HACKATHON_STARTS_AT"]` to the correctly offset iso8601 time. `2017-03-05T07:00:00-05:00`.

**Comment by stevenpetryk** [comment]
I'm at work, so could you give that a try and see if it works?

**Comment by maxcell** [comment]
Absolutely! I tried the config one and that is what sparked my initial hesitation because its time changed from 02:00AM to 10:00PM (and that wasn't even the right time). However, I will keep you posted!

**Comment by maxcell** [comment]
Your iso8601 was the fix :smile: will update.

**Comment by maxcell** [comment]
Changed my timezone to verify the change and it looks good to me! Will add the change
image

**Pull request merged**